README.md

```diff
@@ -107,6 +107,80 @@ DaskTaskRunner(
 )
 ```
 
+### Distributing Dask collections across workers
+
+If you use a Dask collection, such as a `dask.DataFrame` or `dask.Bag`, to distribute the work across workers and achieve parallel computations, use one of the context managers `get_dask_client` or `get_dask_async_client`:
+
+```python
+import dask
+from prefect import flow, task
+from prefect_dask import DaskTaskRunner, get_dask_client
+
+@task
+def compute_task():
+    with get_dask_client() as client:
+        df = dask.datasets.timeseries("2000", "2001", partition_freq="4w")
+        summary_df = df.describe().compute()
+    return summary_df
+
+@flow(task_runner=DaskTaskRunner())
+def dask_flow():
+    prefect_future = compute_task.submit()
+    return prefect_future.result()
+
+dask_flow()
+```
+
+The context managers can be used the same way in both `flow` run contexts and `task` run contexts.
+
+!!! warning "Resolving futures in sync client"
+    Note, by default, `dask_collection.compute()` returns concrete values while `client.compute(dask_collection)` returns Dask Futures. Therefore, if you call `client.compute`, you must resolve all futures before exiting out of the context manager by either:
+    
+    1. setting `sync=True`
+    ```python
+    with get_dask_client() as client:
+        df = dask.datasets.timeseries("2000", "2001", partition_freq="4w")
+        summary_df = client.compute(df.describe(), sync=True)
```

Comment:
Why does the user need to set `sync=True` when they're getting a sync client?

Reply:
The default for `client.compute` returns a Dask Future, unlike `dask.compute` https://distributed.dask.org/en/stable/api.html#distributed.Client.compute
`List of Futures if input is a sequence, or a single future otherwise`